### PR TITLE
People kept saying crawling under flaps isn't an issue because there are windoors, so this gives windoors damage deflection equivalent to airlocks to make them an actual obstacle.

### PR DESCRIPTION
**Pull request by alexkar598**
Closes #11034 

# General Documentation

### Intent of your Pull Request
You wanna play the game of "oh but theres windoors" then let's make those windoors be able to be an actual obstacle instead of a 15 seconds click until its gone meme.

### Why is this change good for the game?
Prevents greytiders without making flaps uncrawlable because that "unintuitive" or something

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
You now need to deal atleast 21 damage to windoors and 30 damage to secure windors in a single hit to damage them at all

:cl:  
tweak: To match airlocks, you now need to deal atleast 21 damage to windoors in a single hit to damage them at all.
tweak: Secure windoors need 30 damage to match reinforced airlocks.
/:cl:

### CODE REVIEW

**Comment by ToasterBiome** [comment]
Sounds like a good change!

**Comment by Hopekz** [comment]
meme PR part 2

**Comment by maxion12345** [comment]
reinforced windoors? yes.
normal ones? actually just glass, not...needed. That way you can secure your secure areas with secure windoors without removing flapcrawling.

**Comment by alexkar598** [comment]
> 
> 
> reinforced windoors? yes.
> normal ones? actually just glass, not...needed. That way you can secure your secure areas with secure windoors without removing flapcrawling.

Either this is an attempt to subvert the PR into not actually mattering or you are ignorant. The only secure windoors on the station are: The warden desk, the hop desk, the holding cell, all 3 brig cells and the ai core. All the departmental windoors are "normal" windoors including the brig office one. I will not be making this PR pointless by making it only apply to secure windoors.

**Comment by maxion12345** [comment]
Sounds like a mapping issue ngl. Put secured toolbox-proof windoors in the secure areas and keep the normal not-toolbox-proof shit for non secure areas like botany or chef's backroom. Also keeps in line with appearances, most people would rightfully assume they can break a non-reinforced glass panel with a toolbox. And again, most delivery points are in maint, meaning they are behind a door they can't toolbox. You'll need to use tools to even access the delivery point, so it isn't just "left click with toolbox until inside department".

**Comment by MegaEmpirical** [comment]
I'm all for making flaps sturdier.

**Comment by JamieD1** [comment]
Feel this is way too high, sometimes this is the only way antags can get into rooms.
